app/models.py

```diff
@@ -6,6 +6,10 @@
 answerType = Union[List[int], str, None]
 
 
+class Organization(BaseModel):
+    org_name: str
```

Comment:
also add an `id` no?

Reply:
This is Response Model, no. We don't need id in our response, only org_name.

Reply:
Anyways, added in the next PR.